packages/golden-layout/test/helper.js

```diff
@@ -1,6 +1,6 @@
 import $ from 'jquery';
-import GoldenLayout from '../src/index.js';
-import lm from '../src/base.js';
+import GoldenLayout from '../dist/index.js';
+import lm from '../dist/base.js';
```

Comment:
Ahh I think you mentioned this - so we need to run the tests from the pre-built code still yeah? May be worth a comment.

Reply:
Ya I tried to get the karma browserify preprocessors to handle TS, but the tsify plugin is dead as browserify is kind of dead.

The tests could possibly be rewritten to not use global scope, but I'm not quite sure how karma handles esm without browserify or some other preprocessing step. I tried babelify, but wasn't successful with it. Felt it wasn't worth spending more of my time on it

I'll add a comment